client/lib/users/test/fixtures/polling-users.js

```diff
@@ -19,5 +19,5 @@ const updatedUsers = clonedMoreUsers.map( user => {
 
 export default {
 	found: 7,
-	users: Array.concat( usersData.users, updatedUsers ),
+	users: [].concat( usersData.users ).concat( updatedUsers ),
```

Comment:
What's wrong with `Array.concat` here? Maybe it should have been `Array.prototype.concat`? Or `lodash.concat`?

Reply:
`Array.concat` doesn't exist, does it? `concat` is only available on `Array.prototype`.

Reply:
I'm not certain why it was working with babel 6.  `Array.concat` is not a function :)